README.md

```diff
@@ -84,9 +77,8 @@ The `alloc` portion specifies account addresses with prefunded ETH when the Ethe
 
 Then, you can build `geth` and init a new instance of a local, Ethereum blockchain as follows:
 
-    $ make geth
-    $ ./build/bin/geth init /path/to/genesis.json -datadir /path/to/your/datadir
-    $ ./build/bin/geth --nodiscover console --datadir /path/to/your/datadir --networkid 12345
+    $ geth init /path/to/genesis.json -datadir /path/to/your/datadir
```

Comment:
No they're not because you need a locally running geth node upon which you'll depoy the validator registration contract down the line, and it is required for our current system to work. We should keep it until our code changes.

Reply:
Ah! You are right!